src/custom/index.js

```diff
@@ -31,7 +32,6 @@ export const customHandleContractTokens = async (
   custom[`${chainId},${contract}`].fetchContractTokens(
     null,
     chainId,
-    token,
```

Comment:
Is `customHandleContractTokens` used anymore? It was broken due to `token` not existing. I noticed after fixing this that many custom `fetchContractTokens` are broken as well due to the `metadata` key not existing.

Reply:
Yeah I think it might not be used anymore, will check with @georgeroman and if so we will cleanup to avoid confusion.